node/narwhal/src/gateway.rs

```diff
@@ -218,7 +218,7 @@ impl<N: Network> Gateway<N> {
 
     /// The maximum number of duplicates for any particular request.
     fn max_cache_duplicates(&self) -> usize {
-        self.max_committee_size() * self.max_committee_size()
+        self.max_committee_size().pow(2)
```

Comment:
wouldn't the compiler produce the same code?

Reply:
I don't think so, as it would have no way of knowing whether the second call results in the same output

Reply:
can we revert since its the same?

Reply:
I'll double-check it, but I'm pretty sure this is not the same, and the current form causes double calculation

Reply:
I see the concern about possible having a double lookup hit, I think we can keep the `pow`

Reply:
I did a quick measurement of runtime performance of both variants under `--release`, and the current version takes ~500ns (with spikes of 1us) on average on my PC, while this change takes it down to ~250ns; both are very quick with a fresh node with an empty db, but this will become slower over time